src/Console/AutomaticInstallation.php

```diff
@@ -95,7 +95,7 @@ protected function getHelp()
 	bin/console autoinstall --savedb
 		Installs Friendica with environment variables and saves them to the 'config/local.config.php' file
 
-	bin/console autoinstall -H localhost -p 3365 -u user -P passwort1234 -d friendica -U https://friendica.fqdn
+	bin/console autoinstall -H localhost -p 3365 -u user -P password1234 -d friendica -U https://friendica.fqdn
```

Comment:
I'm assuming nothing cares?

Reply:
No, in this case it is a direct translation from German to English, not necessarily a typo, but it's still warranted. 😄

Reply:
A number of the changes in this PR were clearly German equivalents. I think I dropped this item from the previous changeset before I made the original PR (I'm fairly certain Google Sheets suggested it and I decided not to spend the time reasoning through it).